checkstyle/checkStyle.xml

```diff
@@ -27,6 +27,11 @@
         <property name="eachLine" value="true"/>
     </module>
 
+    <module name="RegexpSingleline">
+        <property name="format" value="e\.printStackTrace"/>
+        <property name="message" value="Prohibit invoking e.printStackTrace in source code !"/>
```

Comment:
👍 I think this can be ported to main repo as well

Reply:
I think we have already?

Reply:
No, main repo only prohibits `System.out.println`

Reply:
> 👍 I think this can be ported to main repo as well

Got it! Is it appropriate to add the following rules to the main repo too? @kezhenxu94 @wu-sheng @dmsolr 
- Checks that chosen statements are not line-wrapped. By default this Check restricts wrapping import and package statements.
- Checks that non-whitespace characters are separated by no more than one whitespace. Separating characters by tabs or multiple spaces will be reported.

Reply:
> - Checks that chosen statements are not line-wrapped. By default this Check restricts wrapping import and package statements.

Require line wrap in `import` statement is good as that's our current style in main repo but it seems we don't have line wrap before `package`, if that doesn't require too many changes I'm +1. 

> - Checks that non-whitespace characters are separated by no more than one whitespace. Separating characters by tabs or multiple spaces will be reported.

This is good to me.

Reply:
These 2 rules look good to me.